{"uuid":"910c20ca-ca76-44cb-8d5a-0578e6c33e1d","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["364de747-9d58-426c-95a0-6fe4f13be144"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"4540eceb-f2fb-44a1-9092-f63397efed27-attachment","type":"image/png"},{"name":"Myscreenshot","source":"e3d1975a-96ea-4fac-865d-1660b705a30c-attachment"}],"parameters":[],"start":1696884653437,"stop":1696884653500},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884653506,"stop":1696884653506},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884653509,"stop":1696884653509},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884653510,"stop":1696884653510}],"start":1696884653428,"stop":1696884653511}